Tabloid/Controllers/PostController.cs

```diff
@@ -23,5 +23,16 @@ public IActionResult Get()
         {
             return Ok(_postRepo.GetAllPublishedPosts());
         }
+
+        [HttpGet("{id}")]
```

Comment:
Does this need auth protection?

Reply:
Would this be covered by the Authorize attribute at the top of the class?

Reply:
Ah! didn't see that, thanks!